.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,43 @@
+---
+name  : Bug report
+about : Report a bug in Shiny.
+---
+
+<!--
+
+Shiny v1.3.2 is now available on CRAN. If you're currently using an older version of Shiny, we would appreciate it if you could update to Shiny v1.3.2 and confirm whether the problem still persists.
+
```

Comment:
It'd be nice not to have to update this after every CRAN release. Perhaps we could say something like:


> Before filing an issue, please make sure your issue has not already been fixed or filed elsewhere. That is, please test your [reproducible example](https://community.rstudio.com/t/shiny-debugging-and-reprex-guide/10001) holds with recent R package version:

```r
# To install latest shiny from CRAN
install.packages("shiny")
# To update all your R packages
update.packages(ask = FALSE, checkBuilt = TRUE)
# please remember to restart R after updating packages
```

Reply:
Unfortunately I can't format it that nicely since the instructional text lives in an HTML comment, but I agree with the spirit of what you propose 👍

Reply:
It would also be good to tell users to restart R before installing/updating packages, because on Windows, upgrading a package can get messed up if the package is currently loaded.

Reply:
Another thought: I think that asking users to update all their packages is a bit too onerous. I think it's better if we take it out, even if it means it results in some false positives.